grc/__init__.py

```diff
@@ -89,6 +89,10 @@ def make_before_request():
     app.register_blueprint(submitAndPay)
     app.add_url_rule('/submit-and-pay', endpoint='index')
 
+    # Submit and pay
```

Comment:
Can you change the comment to "Policies"

Reply:
Good spot - thanks :-)